data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,270 @@ mission "A Sunracer Hovercraft Racing"
 				clear "second place margin"
 				clear "last place margin"
 				clear "hovercraft racing: talked to fan"
+
+
+phrase "three card monte opening"
+	word
+		"You step down from your ship onto the main concourse of the spaceport, "
+		"Strolling past the smoky glow of the spaceport bar, you drift into the main area, "
+		"After a quiet walk around the area, you find yourself back in the heart of the spaceport, "
+		"You descend from your ship to enter the spaceport, "
+	word
+		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
+		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
+		"when a sharp laugh draws your attention to a small card game on a crate. "
+		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
+	word
+		"The dealer looks up and grins. "
+		"The dealer glances up, eyebrows arched. "
+		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "
+		"The dealer finally notices you, and his grin widens. "
+
+phrase "three card monte opening 2"
+	word
+		"He offers an easy challenge: "
+		"He spreads his arms over the cards in invitation: "
+		""
+		"He grandly gestures at the three waiting cards: "
+
+phrase "card anything but an ace"
+	word
+		"king"
+		"queen"
+		"jack"
+		"ten"
+		"two"
+	word
+		" of"
+
+phrase "card suits"
+	word
+		"hearts"
+		"spades"
+		"clubs"
+		"diamonds"
+
+mission "Three Card Monte"
+	minor
+	repeat
+	"offer precedence" 10
+	to offer
+		random < 20
+		random < 2 + 5 * "flagship planet attribute: tourism" + 5 * "flagship planet attribute: urban" + 10 * "flagship planet attribute: pirate" - "Three Card Monte: declined"
+		credits > 1000
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Pirate"
+		not attributes "deep" "station"
+	on offer
+		conversation
+			`${three card monte opening}`
+			`	The dealer tips a weathered hat. "Always a pleasure to see another captain of fortune." His gaze never leaves your wallet.`
+				to display
+					"reputation: Pirate" > 100
+			`	${three card monte opening 2}"The game's simple. Stake 1,000 credits and find the ace to double your money."`
+			action
+				"monte: not scam chance" = "reputation: Pirate" / 10 + 100
+				"monte: not scam chance" >?= 60 # There's always at least a 60% of not being scammed first round, meaning the player always comes out ahead in the first game
+			label "new game"
+			`	The dealer waits expectantly.`
+			choice
+				`	(Put down 1,000 credits to play the game.)`
+					to display
+						credits > 1000
+				`	(Pat my empty pockets. Can't play today.)`
+					decline
```

Comment:
It would also be neat to have a brief stub here where the scammer tells you what a shame it is you can't play or something.